packages/tupaia-web/src/features/Map/utils/useMapOverlayReport.ts

```diff
@@ -0,0 +1,25 @@
+/*
+ * Tupaia
+ *  Copyright (c) 2017 - 2023 Beyond Essential Systems Pty Ltd
+ */
+
+import { useParams } from 'react-router';
+import {
+  useMapOverlays,
+  useMapOverlayReport as useMapOverlayReportQuery,
+} from '../../../api/queries';
+import { useDateRanges } from '../../../utils';
+import { URL_SEARCH_PARAMS } from '../../../constants';
+
+export const useMapOverlayReport = () => {
```

Comment:
To be honest I keep going back and forth on the use of url params in the query hooks. I added this util as a middle ground so that it keeps the query hooks in the api directory simple but also makes the use of map overlay report data more convenient.

Reply:
I think thats a nice idea, especially for ones like this that pass in more than 1 or 2 params